### DISCUSSION TITLE
Create new google drive folder without upload or upload_resumable

### DISCUSSION THREAD

**Discussion by RedbeanGit** [Byron/google-apis-rs#380] [Ideas]
# What's the problem

Currently, to send `FileCreateCall` request, you have to call `upload()` or `upload_resumable()`. However, you don't always want to send file content, especially when you want to create a new folder.

# Proposal

Maybe it could be good to implement a `doit_without_upload()` function for `FileCreateCall` and make it public.

**Comment by Byron** [comment]
Indeed, some APIs have never been used by me and the generator might produce less convenient APIs for some because of that.

That said, quality of life improvements for `drive3` would be welcome! Those could be implemented with custom settings just for specific APIs that need additional methods to function more conveniently as well - it appears that [one special case](https://github.com/Byron/google-apis-rs/blob/main/etc/api/drive/v3/drive-api_overrides.yaml#L1:L1) is already present, and more could be added and possibly implemented for it. Maybe what's there already supports what you are looking for.
